keg_elements/forms/__init__.py

```diff
@@ -200,6 +201,26 @@ def selected_choice_label(self):
         return value_dict.get(self.data)
 
 
+class RequiredRadioBooleanField(wtforms.fields.RadioField):
```

Comment:
See comments in PR.  I think this should be named `BooleanRadioField` or `BoolRadioField`.